### ISSUE TITLE
Does not load the config file

### ISSUE THREAD

**Issue by xorbuht** [fuyufjh/heygpt#13]
I am on Arch Linux. I've created a config file:
```bash
cat $HOME/.heygpt.toml
model = "gpt-4"
api_base_url = "https://api.openai.com/v1"
api_key = "<myapikeyhere>"
```
However when I run heygpt I see error message "Error: OPENAI_API_KEY not set". It seems heygpt does not load the config file.
How do I make heygpt see my config file?

**Comment by liby** [comment]
[This should be because version 0.4.0 was not released to Crates.io.](https://crates.io/crates/heygpt)

**Comment by fuyufjh** [comment]
Just published 0.4.0 to crates.io. Sorry for my mistake.
